src/index.js

```diff
@@ -0,0 +1,17 @@
+import { handleReset, initialize } from "./display";
+import getCityWeather from "./fetch";
+
+function component() {
+  initialize();
+  const getFormData = () => {
+    handleReset();
+    const temp = document.getElementById("city").value;
+    temp.trim();
+    const city = temp.charAt(0).toUpperCase() + temp.slice(1);
+    const unit = document.getElementById("temp").value;
+    getCityWeather(city, unit);
+  };
+  document.getElementById("submit").addEventListener("click", getFormData);
```

Comment:
Some usability advice: 
It would be better here instead to listen to the submit event on the form element. This way I could submit my input with both the enter key and the button click or even by script. Now I can submit it only by clicking the button, hitting enter refreshes the page. Take a look at this [doc](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event) to check how this can be implemented.